Marlin/src/module/probe.cpp

```diff
@@ -1010,6 +1010,9 @@ float Probe::probe_at_point(const_float_t rx, const_float_t ry, const ProbePtRai
 
     // If any error occurred stow the probe and set an alert
     if (isnan(measured_z)) {
+      // TODO disable steppers ; something defintiely went wrong at this point, so we don't want to insist
```

Comment:
A better TODO would be to make this process recoverable like with Prusa's firmware.

Reply:
> Marlin will completely halt your machine.

apparently only if `G29_HALT_ON_FAILURE` is set (and by default it's not unless `G29_RETRY_AND_RECOVER` is set)

> A better TODO would be to make this process recoverable like with Prusa's firmware.

provided `G29_RETRY_AND_RECOVER`is set, I agree with you. otherwise I still think it should "become soft" (and not halt unless `G29_HALT_ON_FAILURE` is set

Reply:
"become soft" as if your muscles all let go and you offer no physical resistance to your surroundings